assets/aml-benchmark/components/src/aml_benchmark/benchmark_result_aggregator/main.py

```diff
@@ -96,11 +116,11 @@ def _get_run_params(run_definition: Optional[Dict[str, Any]]) -> Dict[str, Any]:
     """
     if run_definition is None:
         return {}
-    node_params = run_definition.get('environmentVariables', None)
+    node_params = run_definition.get("environmentVariables", None)
```

Comment:
how does quotes make a difference? Any issues observed in parsing?

Reply:
This is from black formatter.